recipe/meta.yaml

```diff
@@ -22,17 +22,18 @@ requirements:
     - wheel
```

Comment:
Was [this](https://github.com/jupyter-widgets/ipywidgets/blob/8.1.2/python/ipywidgets/pyproject.toml#L4) intentionally left off because we don't like pinning the build tools?

Reply:
yes, we could add it, but v40 is very old